### PR TITLE
SHANK-239 Updated fall risk model - Disabled IT until we have minimart updated

### PR DESCRIPTION
**Pull request by dlennon-va**
* This is an intermediate PR, we need the model updates in minimart to complete the task.

### CODE REVIEW

File: fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java [hunk 1]
```diff
@@ -70,9 +69,9 @@ private long asEpoch(String time) {
    */
   @GetMapping(params = {"patient"})
   public FallRiskAssessmentResponse searchByPatient(@RequestParam("patient") String patientIcn) {
-    return surveyRepository
-        .findByPatientFullIcnAndSurveyName(patientIcn, FALL_RISK_SURVEY_NAME)
-        .asDatamartSurvey()
+    return fallRiskRepository
```
Comment (jhulbert-va) [inline fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java]: This is a 'search', so it should be returning a list.

File: fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java [hunk 2]
```diff
@@ -70,9 +69,9 @@ private long asEpoch(String time) {
...
+        .findByPatientFullIcn(patientIcn)
+        .asDatamartFallRisk()
```
Comment (jhulbert-va) [inline fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java]: What if this query returns more than 1 record? If so, this gon' break.

File: fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java (hunk 1 above)
Comment (dlennon-va) [reply fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java]: Listed!

File: fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java (hunk 2 above)
Comment (dlennon-va) [reply fall-risk/src/main/java/gov/va/api/health/fallrisk/service/controller/FallRiskAssessmentController.java]: Now returns list and mapped back.

**Pull request merged**